pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -197,11 +214,14 @@ func evictPodsFromTargetNodes(client clientset.Interface, evictionPolicyGroupVer
 		klog.V(3).Infof("evicting pods from node %#v with usage: %#v", node.node.Name, node.usage)
 		currentPodsEvicted := nodepodCount[node.node]
 
+		nonRemovablePods, bePods, bPods, gPods := clasifyPods(node.allPods, evictLocalStoragePods)
```

Comment:
typo in function name, should be `classifyPods`

Also, could you change these to better variable names and comment on `classifyPods` how they are classified? `bePods` can be `bestEffortPods`, same with burstable. I'm not sure what `gPods` stands for though

Reply:
DOne